src/extensions/multiple-sort/locale/bootstrap-table-multiple-sort-es-ES.js

```diff
@@ -0,0 +1,56 @@
+(function (global, factory) {
```

Comment:
@misteregis can you help to update to `src/locale/bootstrap-table-es-ES.js`
like this: https://github.com/wenzhixin/bootstrap-table/blob/develop/src/locale/bootstrap-table-de-DE.js#L119

Reply:
Done!

#7118